pkg/apis/environment/environment.go

```diff
@@ -18,13 +19,13 @@ func CreateEnvironment(pid string, request models.CreateEnvironmentRequest, cred
 	gqlReq.Variables.ProjectId = pid
 	gqlReq.Variables.Request = request
 
-	query, err := json.Marshal(gqlReq)
+	query, _ := json.Marshal(gqlReq)
```

Comment:
instead of replacing it with '_' , it will be better to have a not nil err check here.